source/hardware/controllers/denon_mc7000.rst

```diff
@@ -201,11 +205,15 @@ Deck Section
    "42", ":hwlabel:`SHIFT` + :hwlabel:`X2`",               "Press to create a loop out point at the current Location."
    "43", ":hwlabel:`</> PARAM 1 2`",                       "Press to add/remove rating stars to the loaded track."
    "43", ":hwlabel:`SHIFT` + :hwlabel:`</> PARAM 1 2`",    "Press to change the track color in the library."
-   "44", ":hwlabel:`SLIP`",                                "Press to enable or disable Slip Mode. In Slip Mode, you can jump to cue points, trigger loops or use the platters, while the track’s timeline continues. In other words, when you deactivate Slip Mode, the track will resume normal playback from where it would have been if you had never done anything (i.e., as if the track had been playing forward the whole time)."
+   "44", ":hwlabel:`SLIP`",                                "Press to enable or disable Slip Mode. In Slip Mode, you can jump to cue points, trigger loops or use the platters, while the track’s timeline continues. In other words, when you deactivate Slip Mode, the track will resume normal playback from where it would have been if you had never done anything (i.e., as if the track had been playing forward the whole time).
+
+   After scratch, hot cue play, reverse play and backspin the slip mode releases itself so the timeline of the track jumps back to original position after the action is done. Slip mode reactivates itself after 50ms so you can continue with more actions in slip mode almost immediately after."
```

Comment:
This sounds like an interesting feature!

Reply:
Yes it is. There was a discussion on [zulip chat](https://mixxx.zulipchat.com/#narrow/stream/109122-general/topic/Slip.20Mode/near/240571349) that triggered me to add it to the MC7000 mapping. Short timer after the release to enable SLIP again. 50ms seems to be very good. Short enough you could not press a button (incl platter) twice that quickly. Long enough not to miss something. The 50ms is a pure empiric value though